### DISCUSSION TITLE
Any chance to get openmptcprouter installed on a GL-X1200 (Amarok)?

### DISCUSSION THREAD

**Discussion by robmicro** [Ysurac/openmptcprouter#2170] [Q&A]
this router has two mini PCIe modules for 4G LTE. would be perfect for mptcp

**Comment by packlet** [comment]
I think it would make a nice ROOter, but you'd probably need more processor (like in the AP1300/S1300) for heavy VPN activity needed...

And while gl.inet generally is openwrt-friendly, there's (yet) no listing for the x1200 here: https://openwrt.org/toh/views/toh_vpn_performance?dataflt%5BBrand*%7E%5D=gl.inet&datasrt=cpu+mhz&dataflt%5BModel*%7E%5D=
